modules/dumbCommandModule/index.js

```diff
@@ -74,13 +74,13 @@ module.exports.commands = {
       reply(`Blame ${commandDict[parts[0]].blame} for this`);
     },
     rblame(r, parts, reply) {
-      if (parts.length === 0) return reply('please specify a user to blame');
+      if (parts.length !== 0) return reply('please specify a user to blame');
```

Comment:
I'm an idiot but it doesn't matter because I changed this line in the lint commit